features/serializers.py

```diff
@@ -13,6 +13,7 @@ class FindSimilarSerializer(serializers.Serializer):
     language = serializers.CharField(required=False)
     count = serializers.IntegerField(default=5)
     dictionary = serializers.DictField(required=False)
+    remove_stopwords = serializers.BooleanField(required=False)
```

Comment:
I haven't seen that GitHub action with tests was run. Was all test passed and coverage wasn't changed?

Reply:
![image](https://github.com/findsimilar/demo/assets/112252001/3313cd47-83d7-4a63-ac8c-09ff668802f9)